src/common/NavigationConstants.js

```diff
@@ -16,7 +16,8 @@ export default {
     LOGIN: "login",
     LOGOUT: "logout",
     MORE: "more",
-    NEW_APP: "apps/create",
+    NEW_APP: "/apps/create",
+    NEW_WORKFLOW: "/pipelines/create",
```

Comment:
This allows the user to navigate to the create page from an app-by-id listing page, such as https://de.cyverse.org/apps/de/67d15627-22c5-42bd-8daf-9af5deecceab.

Reply:
I was only mirroring the API endpoints, which use `/apps/pipelines`, but I'm fine with sticking with `/workflows` in the UI.